shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -220,14 +220,16 @@ void PostAccessibilityNotification(UIAccessibilityNotifications notification,
     }
 
     if (layoutChanged) {
-      ios_delegate_->PostAccessibilityNotification(UIAccessibilityLayoutChangedNotification,
-                                                   FindNextFocusableIfNecessary());
+      ios_delegate_->PostAccessibilityNotification(
+          UIAccessibilityLayoutChangedNotification,
+          [FindNextFocusableIfNecessary() nativeAccessibility]);
```

Comment:
It would be nice if we had tests that asserted nativeAccessibility is called here.